src/main/scala/tilelink/Parameters.scala

```diff
@@ -534,48 +537,81 @@ class TLSlavePortParameters private(
   // Does this Port manage this ID/address?
   def containsSafe(address: UInt) = findSafe(address).reduce(_ || _)
 
-  private def supportHelper(
+  private def addressHelper(
       safe:    Boolean,
```

Comment:
`// setting safe to false indicates that all addresses are expected to be legal, which might reduce circuit complexity`